tests/python/gaia-ui-tests/gaiatest/apps/messages/regions/new_message.py

```diff
@@ -27,7 +27,7 @@ class NewMessage(Messages):
 
     def __init__(self, marionette):
         Base.__init__(self, marionette)
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == self.name)
+        Wait(self.marionette).until(lambda m: self.apps.displayed_app.manifest_url == self.manifest_url)
```

Comment:
I didn't see that duplication until today. I would factorize this wait in the base class:
``` python
class Base:
    def wait_to_be_displayed():
        Wait(self.marionette).until(lambda m: self.apps.displayed_app.manifest_url == self.manifest_url)
```

And call this function in the inherited classes. So, next time we change the way we wait for that, we'd have to change only one line, instead of 10.